mlflow/tracking/client.py

```diff
@@ -483,6 +483,7 @@ def get_trace(self, request_id: str, display=True) -> Trace:
     def search_traces(
         self,
         experiment_ids: List[str],
+        run_id: Optional[str] = None,
```

Comment:
Can we move this argument to the end as well? https://github.com/mlflow/mlflow/pull/13251/files#r1777536058